objstore.go

```diff
@@ -692,7 +705,10 @@ func (rc *timingReadCloser) Close() error {
 
 func (rc *timingReadCloser) Read(b []byte) (n int, err error) {
 	n, err = rc.ReadCloser.Read(b)
-	rc.fetchedBytes.WithLabelValues(rc.op).Add(float64(n))
+	if rc.fetchedBytes != nil {
```

Comment:
When would `fetchedBytes` be nil?

Reply:
When we use the newTimingReadCloser structure in the Upload method:
trc := newTimingReadCloser(
		io.NopCloser(r),
		op,
		b.opsDuration,
		b.opsFailures,
		b.isOpFailureExpected,
		nil,
		b.opsTransferredBytes,
	)